framework/requirements/requirements.php

```diff
@@ -5,53 +5,53 @@
  */
 
 /* @var $this YiiRequirementChecker */
-return array(
-    array(
+return [
+    [
         'name' => 'PHP version',
         'mandatory' => true,
         'condition' => version_compare(PHP_VERSION, '5.4.0', '>='),
         'by' => '<a href="http://www.yiiframework.com">Yii Framework</a>',
         'memo' => 'PHP 5.4.0 or higher is required.',
```

Comment:
I think this replacement should not be done for this file. There is a small chance of running this on PHP < 5.4.0 where short array syntax is not yet supported. While it's OK for other core framework files because Yii 2 requires PHP >= 5.4.0 and it's mentioned in the docs, requirements checker has different purpose, it executes separately and out of framework's scope, before its installation. So for earlier versions of PHP instead of showing "PHP 5.4.0 or higher is required." message it will simply fail with parsing square brackets error.